docs/apache-airflow/extra-packages-ref.rst

```diff
@@ -343,27 +350,278 @@ you need to list explicitly all the non-bundle extras that you want to install.
 +---------------------+-----------------------------------------------------+------------------------------------------------------------------------+
 | all_dbs             | ``pip install 'apache-airflow[all_dbs]'``           | All database integrations                                              |
```

Comment:
Maybe not really/fully relating to this PR but just "stumbling" over this: What is "All database" referring to? Is there a list of "all"? Referring to provider packages "smelling" like DB or all database types we support in Airflow for the Airflow meta data (a.k.a sqlite, postgres, mysql/maria nowadays)

Reply:
I **think** it's had some use - but I have no idea what kind of use it would be. I am mostly keeping it for backwards compatibility - but it was there defined when I started contributing to Airflow and I am not sure if there is anyone any more who would know the context. 

I'd be happy to remove it, if others think it's safe to do so and there is no use that they can think of.

Let's leave it open for others who might want to comment on it.

Reply:
This can be resolved later.